WowPacketParser/Store/Objects/TrainerSpell.cs

```diff
@@ -6,6 +6,15 @@ namespace WowPacketParser.Store.Objects
     [DBTableName("trainer_spell")]
     public sealed class TrainerSpell : IDataModel
     {
+        public uint[] ReqAbility;
+
+        public void ConvertToDBStruct()
```

Comment:
why is this needed? `DBFieldName` can be configured with `startAtZero`

Reply:
Otherwise it puts tons of UPDATES even if they are not necessary.